packages/protocol/src/data/inscription-type.ts

```diff
@@ -21,15 +21,3 @@ export namespace ActivityEditorType {
   export type Bpmn = 'GenericBpmnElement' | 'UserBpmnElement' | 'ManualBpmnElement' | 'ScriptBpmnElement' | 'ServiceBpmnElement' | 'RuleBpmnElement' | 'SendBpmnElement' | 'ReceiveBpmnElement';
   export type All = General | Interface | Bpmn;
 }
-
-export type OtherEditorType = 'ProcessAnnotation' | 'Unknown';
-
-export type InscriptionEditorType = EventEditorType.All | GatewayEditorType | ActivityEditorType.All | OtherEditorType;
-
-export interface InscriptionTypeBeta {
```

Comment:
the idea of this PR is mainly, to get rid of this re-declaration : as the inscriptionType with its valid element-Types already exists out of the generated inscription.ts protocol